assets/scripts/books/ui.js

```diff
@@ -5,7 +5,19 @@ const showBooksTemplate = require('../templates/book-listing.handlebars');
 const getBooksSuccess = (data) => {
   console.log(data);
   let showBooksHtml = showBooksTemplate({books: data.books});
+  // $("button").on("click", function(e){
+  //   e.preventDefault();
+  //   $(e.target).parent().parent().remove();
+  // })
+  // $("#content").on("click", 'button', function(e){
+  //   e.preventDefault();
+  //   $(e.target).parent().parent().remove();
+  // })
   $('.content').append(showBooksHtml);
+  // $("button").on("click", function(e){
+  //   e.preventDefault();
+  //   $(e.target).parent().parent().remove();
+  // })
```

Comment:
Missing semi colons on the solution, but I think we can open that as a post 016 issue. The solution is working when I uncomment it and test it out, it just comes with some linter warnings.